README.md

```diff
@@ -38,11 +38,10 @@ main();
 
 ## License
 
-@foxglove/electron-socket is released under the [MIT License](/LICENSE.md).
+@foxglove/electron-socket is licensed under [MIT License](https://opensource.org/licenses/MIT).
```

Comment:
linking to our actual license seems more sensible?

Reply:
That link doesn't work when published to npm. I think that's why our other repos are using this pattern.

Reply:
ah good point, didn't think of that